content/en/docs/components/katib/experiment.md

```diff
@@ -379,18 +401,19 @@ Katib supports the following algorithm settings:
 </div>
 
 <a id="enas"></a>
+
 #### Neural Architecture Search based on ENAS
 
 {{% alert title="Alpha version" color="warning" %}}
-Neural architecture search is currently in <b>alpha</b> with limited support. 
-The Kubeflow team is interested in any feedback you may have, in particular with 
+Neural architecture search is currently in <b>alpha</b> with limited support.
```

Comment:
```suggestion
Neural architecture search (NAS) is currently in <b>alpha</b> with limited support.
```

although it must've been predefined but just in case

Reply:
Defined here: https://github.com/kubeflow/website/blame/760ec66a6a2327d0b9ae814fa7695dbbb3d537a5/content/en/docs/components/katib/experiment.md#L10.